relay-metrics/src/aggregation.rs

```diff
@@ -3098,36 +3138,36 @@ mod tests {
         // be processed, so we need to convert them to a set:
         let (partition_keys, tail) = output.split_at(2);
         insta::assert_debug_snapshot!(BTreeSet::from_iter(partition_keys), @r###"
-        {
-            "metrics.buckets.partition_keys:59|h",
-            "metrics.buckets.partition_keys:62|h",
-        }
-        "###);
+         {
+             "metrics.buckets.partition_keys:59|h",
+             "metrics.buckets.partition_keys:62|h",
+         }
+         "###);
 
         insta::assert_debug_snapshot!(tail, @r###"
-        [
-            "metrics.buckets.per_batch:1|h",
-            "metrics.buckets.batches_per_partition:1|h",
-            "metrics.buckets.per_batch:1|h",
-            "metrics.buckets.batches_per_partition:1|h",
-        ]
-        "###);
+         [
+             "metrics.buckets.per_batch:1|h",
+             "metrics.buckets.batches_per_partition:1|h",
+             "metrics.buckets.per_batch:1|h",
+             "metrics.buckets.batches_per_partition:1|h",
+         ]
+         "###);
     }
 
     fn test_capped_iter_completeness(max_flush_bytes: usize, expected_elements: usize) {
         let json = r#"[
-          {
-            "name": "endpoint.response_time",
-            "unit": "millisecond",
-            "value": [1, 1, 1, 1],
-            "type": "d",
-            "timestamp": 1615889440,
-            "width": 10,
-            "tags": {
-                "route": "user_index"
-            }
-          }
-        ]"#;
+           {
+             "name": "endpoint.response_time",
+             "unit": "millisecond",
+             "value": [1, 1, 1, 1],
+             "type": "d",
+             "timestamp": 1615889440,
+             "width": 10,
+             "tags": {
+                 "route": "user_index"
+             }
+           }
+         ]"#;
```

Comment:
this is all pretty noisy, is rustfmt no longer happy with the previous formatting?

Reply:
so, that what `cargo fmt` did.